FlowCrypt/Functionality/PhotosManager/PhotosManager.swift

```diff
@@ -10,10 +10,12 @@ import Foundation
 import UIKit
 import Photos
 import Combine
+import PhotosUI
 
 typealias PhotoViewController = UIViewController
 & UIImagePickerControllerDelegate
 & UINavigationControllerDelegate
```

Comment:
`UIImagePickerControllerDelegate` and `UINavigationControllerDelegate` not needed here too

Reply:
done